packages/edit-site/src/components/page-templates/fields.js

```diff
@@ -155,6 +155,11 @@ export const activeField = {
 	id: 'active',
 	getValue: ( { item } ) => item._isActive,
 	render: function Render( { item } ) {
+		if ( item.is_custom ) {
+			// translators: Refers to a type of template: custom template
+			return <Badge intent="info">{ __( 'Custom' ) }</Badge>;
+		}
```

Comment:
As a site admin, I might not fully understand what “Custom” means when it appears next to the status. I’d associate “Custom” with a template type instead, which is also pixels away from the status, and could feel redundant.

My expectation would be to see the actual status (active or inactive) and, if the template type is custom, get some context on why I can or can’t make changes.

Maybe a padlock icon on the active/inactive status for custom templates, with a tooltip explaining the limitation, could work well here.

Also, labeling it as a “custom status” might suggest it’s something I can create or edit, which could be confusing.

Reply:
Thanks for the feedback, @juanfra!

My initial version had a more basic "N/A" label instead. Because custom templates really don't have a status. At least right now they don't; maybe at some point we could attempt to track them to posts that use them, but none of that is built.

> padlock icon

From a design / components standpoint, I don't know if we can do this, I'd like input from designers.

> with a tooltip explaining the limitation, could work well here

This makes perfect sense, I think we should. However, I'm working with @ellatrix and we'd like to merge something imperfect _now_ and polish those interactions during the Beta cycle.

I'll quickly address the `_x()` part, but I hope you can approve this more or less as-is. Yay or nay on the "N/A" label?

Reply:
Thanks for the response, Miguel! I understand the urgency with 6.9 around the corner, so I agree that “N/A” works better than the “Custom” approach. I’m good approving this as-is, with the hope that we can revisit and refine it in the future.